### ISSUE TITLE
Allow ignore specific action

### ISSUE THREAD

**Issue by yesmeck** [maxmechanic/redux-fsa-linter#1]
It would be nice if can ignore some external library actions:

``` javascript
createLinter({
  ignore: [
    /redux-form\/.+/,
    /@reudx-modal\/.+/,
  ]
})
```

**Comment by maxmechanic** [comment]
I've thought about this a little, and am opting for [this approach](https://github.com/maxmechanic/redux-fsa-linter/commit/b0899f9ca3a8da63c00e45712aea750611c4f668). 

You will be able to pass a predicate function to `createLinter` to test for what should be ignored:

``` js
createLinter({
  ignore: ({ type }) => type.match(/redux-form\/.+/)
})
```

The predicate is just handed the whole action, so you can adapt the function to whatever you expect from those actions.

I need to patch the README, after which I will version-bump and publish to npm.

**Comment by maxmechanic** [comment]
Closed by #2
